public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace draw_line_on_window
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create Window
+            Window start = new Window("draw line on window", 600, 600);
+            SplashKit.ClearScreen(SplashKit.ColorBlack());
+
+
+            // Draw line on window - param (Window, Color, x1, y1, x2, y2)

```

Comment:
Improve this comment to be about what the code is doing (overall image rather than just the function name and parameters, as these are already shown below).